backend/model/db.js

```diff
@@ -88,5 +100,7 @@ module.exports = {
   Department,
   Title,
   Follow,
-  CollabRequest
+  CollabRequest,
+  Notification,
+  Event
```

Comment:
always add comma (",") after the last element of object

Reply:
sure